setup.py

```diff
@@ -6,44 +6,39 @@
 from setuptools import setup, find_packages
 
 # Read in the __version__ attribute without importing the package
-with open('yinsolidated/_version.py') as version_file:
+with open("yinsolidated/_version.py") as version_file:
     exec(version_file.read())
 
 setup(
-    name='yinsolidated',
+    name="yinsolidated",
     version=__version__,
     description=(
-        'Generates and parses a YIN-like representation of a YANG model, '
-        'consolidated into a single XML document'
+        "Generates and parses a YIN-like representation of a YANG model, "
+        "consolidated into a single XML document"
     ),
-    url='https://github.com/128technology/yinsolidated',
-    author='Alex Thompson',
-    author_email='alex@128technology.com',
-    license='MIT',
+    url="https://github.com/128technology/yinsolidated",
+    author="Alex Thompson",
+    author_email="alex@128technology.com",
```

Comment:
🤔

Reply:
who dat?

Reply:
who wants it? Hadriel, you want to be the author of this package?

Reply:
hell no - find someone who's not paying attention to the PR right now 😄